docs/tutorial/configurations/modelview/index.tr.md

```diff
@@ -0,0 +1,224 @@
+
+Şablon dosyaları, `Jinja2` kullanılarak oluşturulmuştur ve konfigürasyonlarda tamamen değiştirilebilir. Kullanılabilir sayfalar:
+
+* `list_template`: Liste görünümü şablonu. Varsayılan değer `list.html` olarak ayarlanmıştır.
+* `detail_template`: Detay görünümü şablonu. Varsayılan değer `detail.html` olarak ayarlanmıştır.
+* `create_template`: Oluşturma görünümü şablonu. Varsayılan değer `create.html` olarak ayarlanmıştır.
+* `edit_template`: Düzenleme görünümü şablonu. Varsayılan değer `edit.html` olarak ayarlanmıştır.
+
+!!! example "Örnek"
+
+    ```Python
+    class PostView(ModelView):
+        detail_template = "post_detail.html"
+    ```
+
+## Datatables Extensions - Datatables Eklentileri
+
+*starlette-admin* ile birlikte bazı datatable eklentileri varsayılan olarak gelir. Bu eklentilerden herhangi birini `ModelView`'inizde aşağıdaki seçenekleri geçersiz kılarak devre dışı bırakabilirsiniz:
+
+* `column_visibility`: [column visibility](https://datatables.net/extensions/buttons/built-in#Column-visibility) eklentisini etkinleştir/devre dışı bırak
+* `search_builder`: [search builder](https://datatables.net/extensions/searchbuilder/) eklenini etkinleştir/devre dışı bırak
+* `responsive_table`: [responsive](https://datatables.net/extensions/responsive/) eklentisini etkinleştir/devre dışı bırak
+* `save_state`: [state saving](https://datatables.net/examples/basic_init/state_save.html) eklentisini etkinleştir/devre dışı bırak
+
+!!! example "Örnek"
+
+    ```python
+    class PostView(ModelView):
+        column_visibility = False
+        search_builder = False
+        responsive_table = True
+        save_state = True
+    ```
+
+## Object Representation - Nesne Temsili
+
+*sarlette-admin* yönetici arayüzünde nesnelerin nasıl temsil edileceğini özelleştirmek için iki yöntem mevcuttur:
+
+### `__admin_repr__`
+
+Yönetici arayüzünde nesnelerin nasıl temsil edileceğini özelleştirmek için model sınıflarında kullanılabilen özel bir metoddur. Varsayılan olarak yalnızca nesnenin birincil anahtar özelliğinin değeri görüntülenir. Bununla birlikte, `__admin_repr__` metodunu kullanarak, nesneyi yönetici arayüzünde daha iyi temsil edebilecek bir `str` döndürebilirsiniz.
+
+!!! example "Örnek"
+    Aşağıdaki `User` modeli için örnek uygulama, yönetici arayüzünde kullanıcının birincil anahtar yerine tam adını gösterir:
+
+    ```python
+    class User:
+        id: int
+        first_name: str
+        last_name: str
+
+        async def __admin_repr__(self, request: Request):
+            return f"{self.last_name} {self.first_name}"
+    ```
+
+    ![Custom Object representation](../../../images/tutorial/configurations/modelview/object_text_representation.png){ width="200" }
+
+### `__admin_select2_repr__`
+
+`__admin_repr__` metoduna benzeyen bu metod, nesneyi `select2` bileşeninde görüntülemek için kullanılabilecek bir HTML döndürür. Varsaylına olarak nesnenin tüm özellikleri kullanılır, ancak ilişki ve dosya alanlarını içermez.
```

Comment:
```suggestion
`__admin_repr__` metoduna benzeyen bu metod, nesneyi `select2` bileşeninde görüntülemek için kullanılabilecek bir HTML döndürür. Varsayılan olarak, nesnenin, detay sayfası için izin verilen tüm özellikleri kullanılır ancak ilişki ve dosya alanlarını içermez.
```

Reply:
Burayı context dışında çevirmişsin, detay sayfası için değil orası. Select2 içerisinde ekleme ve düzenleme sayfalarında görüntülemek için bir HTML şablonu yazıyorsun aslında orda.